### ISSUE TITLE
Typhoon not compiling after update pod

### ISSUE THREAD

**Issue by jcmartinac** [appsquickly/Typhoon#542]
I have updted my pods, and typhoon has changed version from 3.5.1 to 4.0.1, and compiling throws this: 
<img width="256" alt="captura de pantalla 2017-01-20 a las 8 50 42" src="https://cloud.githubusercontent.com/assets/6225136/22140586/93f719ec-deed-11e6-8cab-18bcc97bb89e.png">

I have done a clean and clean build folder without solve it. Any clue?

**Comment by alexgarbarev** [comment]
Yeah, I think something is not right inside your Pods configuration. Try to delete Pods folder and then call `pod install`. Also make sure that Typhoon only added to your main project target (not tests target)

**Comment by jcmartinac** [comment]
Following your directions I got same error. This is mi pod file:

source 'https://github.com/CocoaPods/Specs.git'

use_frameworks!

platform :ios, '9.0'

abstract_target 'Project' do
    pod 'AFNetworking', '~> 3.0'
    pod 'Typhoon'
    pod 'JSONModel'
    pod 'Realm'
    pod 'Crashlytics'
    pod 'OHHTTPStubs'
    pod 'EXTView'
    pod 'ActionSheetPicker-3.0'
    
    target 'Project-PRO'
    target 'Project-QA'
    target 'Project-DEV'
    target 'Project-AppStore'
    
end

target 'ProjectTests' do
    
    pod 'OCMock'
    pod 'OHHTTPStubs'
    pod 'Realm'
    
end

target 'ProjectIntegrationTests' do
    
    pod 'OCMock'
    pod 'OHHTTPStubs'
    pod 'Realm'
    
end

**Comment by etolstoy** [comment]
@jcmartinac do you still have this problem?

**Comment by jcmartinac** [comment]
@etolstoy nothing change since last time... I am working with version: pod 'Typhoon', '~> 3.5.1', because I don't know how to solve it

**Comment by etolstoy** [comment]
@jcmartinac I'll check it.

**Comment by jcmartinac** [comment]
@etolstoy did you find anything?

**Comment by cbedoy** [comment]
I'm having the same issue but with xcode 8.3

**Comment by AdithyaReddy** [comment]
<img width="428" alt="screen shot 2018-07-02 at 1 37 47 pm" src="https://user-images.githubusercontent.com/3955341/42152255-30fd4928-7dfd-11e8-9722-79dfd4d34274.png">
I am trying to migrate from 3.5.1 -> 4.0.5 and I get the above error. My codebase is running on Swift3.2.
